src/Type/Php/SprintfFunctionDynamicReturnTypeExtension.php

```diff
@@ -132,14 +136,14 @@ public function getTypeFromFunctionCall(
 								continue 2;
 							}
 						}
-						$singlePlaceholderEarlyReturn = TypeCombinator::union(...$result);
+						$singlePlaceholderEarlyReturn[] = TypeCombinator::union(...$result);
```

Comment:
Would be better to do one union at the end instead of these intermediate ones.

Reply:
Indeed ; fixed.